src/assets/stylesheets/shared/_layout.scss

```diff
@@ -1,371 +0,0 @@
-///////////////////////////
-// Utility styles
-///////////////////////////
-
-@mixin padding-x($count:15) {
-  @for $i from 0 through $count {
-    .padding-#{$i} {
-      padding: #{$i}px !important;
-    }
-    .padding-top-#{$i} {
-      padding-top: #{$i}px !important;
-    }
-    .padding-left-#{$i} {
-      padding-left: #{$i}px !important;
-    }
-    .padding-bottom-#{$i} {
-      padding-bottom: #{$i}px !important;
-    }
-    .padding-right-#{$i} {
-      padding-right: #{$i}px !important;
-    }
-  }
-}
```

Comment:
Do we want to remove this or perhaps it has moved?

FYI, I'm using the following Less conversion in the patternfly-ng repo.

.padding-loop (@i) when (@i > 0) {
  .padding-@{i} {
    padding: ~"@{i}px" !important;
  }
  .padding-top-@{i} {
    padding-top: ~"@{i}px" !important;
  }
  .padding-left-@{i} {
    padding-left: ~"@{i}px" !important;
  }
  .padding-bottom-@{i} {
    padding-bottom: ~"@{i}px" !important;
  }
  .padding-right-@{i} {
    padding-right: ~"@{i}px" !important;
  }
  .padding-loop(@i - 1);
}
.padding-loop(15);